src/sass/_common/common.scss

```diff
@@ -674,6 +674,20 @@ ul.bullet.gr-row > li {
     display: inline-block;
 }
 
+.gr-adapt .cashier-button.toggle.button {
+    margin-left: 0;
+    white-space: nowrap;
+}
+
+#demo-btn {
+    margin-left: -1.3rem !important;
```

Comment:
(Optional) I think it’s not the best way to center a button, maybe changing `gr-12-m gr-2` classes on html will help.